### DISCUSSION TITLE
What's in the Octoverse 2023?

### DISCUSSION THREAD

**Discussion by samus-aran** [community/community#74740] [Announcements]
![github-octoverse-hero-image-2023_](https://github.com/community/community/assets/4041275/8524fcab-19d2-45da-b674-eb79d3a02921)
> Alt Text: The image displays text The state of the Octoverse 2023 on a black background and colored shapes in the foreground

The Octoverse is an annual report that looks at the state of open source. Providing insights and data on the GitHub platform, including details about the open-source software community and development trends.

[In this year’s report](https://github.blog/2023-11-08-the-state-of-open-source-and-ai/), we’ll study how open source activity around AI, the cloud, and Git has changed the developer experience and is increasingly driving impact among developers and organizations alike.

Here is a sneak peek of what we have uncovered:

- **Developers are building with generative AI in big numbers** 🤖 
92% of developers are using or experimenting with AI coding tools, so we expect open source developers to drive the next wave of AI innovation on GitHub.

- **Developers are operating cloud-native applications at scale** ☁️ 
We’re seeing an increase in declarative languages using Git-based infrastructure as code (IaC) workflows, greater standardization in cloud deployments, and a sharp increase in the rate at which developers were using Dockerfiles and containers, IaC, and other cloud-native technologies. 

- **2023 saw the largest number of first-time open source contributors 📈** 
We continue to see commercially backed open source projects capture the largest share of first-time contributors and overall contributions—but this year, we also saw generative AI projects enter the top 10 most popular projects for first-time contributors. \

- **Who our fastest growing countries are across the globe 🌐**
With 20.2 million developers and a 21% increase in developer growth over the past year, the U.S. continues to have the largest developer community globally. But since 2013, we’ve continued to see other communities account for more growth across the platform which we expect to continue.** 

- **What are the most popular programming languages**
Spoiler: it isn’t COBOL 💻

Discover even more from our detailed breakdown of Octoverse 2023 in our [blog post](https://github.blog/2023-11-08-the-state-of-open-source-and-ai/). If you’re a visual learner we have prepared a number of [infographics](https://github.blog/2023-11-08-the-state-of-open-source-and-ai/#a-global-community-of-developers-building-on-github) and [videos](https://www.youtube.com/watch?v=cxkJLeGfxyM) to help you digest all the fascinating results. 

<img width="50%" alt="Geographic heatmap of developer populations across the world " src="https://github.com/community/community/assets/4041275/9f8c01b9-c348-4378-acc6-90fbfbf27f80">

> Alt Text: The image displays a geographic heatmap of developer populations across the world

## Join us Live with Martin Woodward at GitHub Universe 2023 🪐

**[Watch us live](https://githubuniverse.com/),** Thursday 9th November, as Martin Woodward, VP Developer Relations at GitHub, dives into the latest State of the Octoverse report. 📺

Which result did you find most surprising? Are there any trends that you predicted 🔮? Share your thoughts on the state of open source and this year’s Octoverse report in the comments below 👇

**Comment by mishmanners** [comment]
So excited to see this published for everyone ❤️

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
